### PR TITLE
Victory and death splash

### PR DESCRIPTION
**Pull request by AngieDutra**
## Motivation

The death splash needed a style refactor

## Summary of changes

This PR changes the style of the death splash and also fixes the killfeed to show the zone or muflus depending on who killed a player

prueba boton 3

defeated-ref

List tests added/updated.

## Checklist
- [x] I have tested the changes locally.
- [x] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [ ] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [ ] Tested in Android.

### CODE REVIEW

File: client/Assets/Scripts/UI/KillFeedItem.cs [hunk 1]
```diff
@@ -10,10 +10,16 @@ public class KillFeedItem : MonoBehaviour
     private const string LOOT_ID = "1111";
 
     [SerializeField]
-    TextMeshProUGUI killerPlayer;
+    TextMeshProUGUI killerPlayer,
+        killedPlayer;
 
     [SerializeField]
-    TextMeshProUGUI killedPlayer;
+    Sprite muflusIcon,
```
[the remark this answers is not in the document]
Comment (AngieDutra) [reply client/Assets/Scripts/UI/KillFeedItem.cs]: Changes on this were pushed

**Review by BertovDev — CHANGES_REQUESTED** [review]
This PR contemplates get killed by the DangerZone? if it does im getting an error when i get kileld by it.
 It is trying to find a player with the dangerZone id. This happends in the method: `GetCharacterUIIcon` and then in `GetDefeaterSprite`

File: client/Assets/Scripts/CustomLevelManager.cs [hunk 2]
```diff
@@ -129,7 +123,11 @@ void Update()
         }
         if (GameHasEnded())
         {
-            deathSplash.GetComponent<DeathSplashManager>().ShowEndGameScreen();
+            // TODO: Redirect to EndGameScreen
+            //SceneManager.LoadScene("EndGame");
+            EndGameManager endGameManager = deathSplash.GetComponentInChildren<EndGameManager>();
```
[the remark this answers is not in the document]
Comment (AngieDutra) [reply client/Assets/Scripts/CustomLevelManager.cs]: done!

File: client/Assets/Scripts/UI/KillFeedManager.cs [hunk 3]
```diff
@@ -36,6 +43,17 @@ public ulong GetKiller(ulong deathPlayerId)
         return killerId;
     }
 
+    Sprite GetCharacterUIIcon(ulong playerId)
+    {
+        CoMCharacter characterIcon = KillFeedManager.instance.charactersScriptableObjects.Single(
+            characterSO =>
+                characterSO.name.Contains(
+                    Utils.GetPlayer(playerId).GetComponent<CustomCharacter>().CharacterModel.name
```
[the remark this answers is not in the document]
Comment (AngieDutra) [reply client/Assets/Scripts/UI/KillFeedManager.cs]: done!

**Review by ncontinanza — APPROVED** [review]
LGTM!

**Pull request merged**